README.md

```diff
@@ -100,7 +115,7 @@ SQL database. If you need an example, you can use: people and wishes
 
 Your answer:
 ```
-Replace this with your answer
+The schema maps out how the how information will be stored into each table.
```

Comment:
:+1: Good, but more detail would be good here, e.g. the schema defines the names of the tables, what columns are in each table (name and data type).